File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.E3Result;
import entity.PageResult;

import java.util.List;
import java.util.Map;

public interface BrandService {

    List<TbBrand> findAll();

    PageResult findPage(int page, int size);

    E3Result add(TbBrand tbBrand);

    TbBrand findOne(Long id);

    E3Result update(TbBrand tbBrand);

    E3Result delete(Long[] ids);

    PageResult findPage(TbBrand tbBrand, int page, int size);

    List<Map> selectOptionList();

}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service("brandService",function ($http) {
    this.findAll=function () {
        return $http.get('../brand/findAll.do');
    }
    this.pageAll=function (page,size) {
        return $http.get('../brand/pageAll.do?page='+page+'&size='+size);
    }
    this.search=function (page,size,searchEntity) {
        return $http.post('../brand/search.do?page='+page+'&size='+size,searchEntity);
    }
    this.add=function (entity) {
        return $http.post('../brand/add.do',entity);
    }
    this.update=function (entity) {
        return $http.post('../brand/update.do',entity);
    }
    this.findById=function (id) {
        return $http.get('../brand/findById.do?id='+id)
    }
    this.del=function (id) {
        return $http.get('../brand/del.do?ids='+id);
    }
    this.selectOptionList=function () {
        return $http.get('../brand/selectOptionList.do');
    }
});